hudi-client/hudi-spark-client/src/main/scala/org/apache/hudi/HoodieDatasetBulkInsertHelper.scala

```diff
@@ -241,17 +241,16 @@ object HoodieDatasetBulkInsertHelper
     }
   }
 
-  private def getPartitionPathFields(config: HoodieWriteConfig): Seq[String] = {
+  private def getPartitionPathFields(config: HoodieWriteConfig): mutable.Seq[String] = {
```

Comment:
There are immutable and mutable `Seq`.  In Scala 2.13, `scala.Seq[+A]` is now an alias for `scala.collection.immutable.Seq[A]` (instead of `scala.collection.Seq[A]`) (see https://docs.scala-lang.org/overviews/core/collections-migration-213.html).  When not necessary, we pass around `mutable.Seq`.